ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -5519,6 +5519,106 @@ presubmits:
           requests:
             cpu: 500m
     trigger: (?m)^/test shellcheck
+  - agent: kubernetes
+    always_run: true
```

Comment:
I think this job should have `always_run: false` and a `run_if_changed` field, right?

Reply:
Do you know where to set that? I couldn't find a trigger-based container image generation example.

Reply:
Isn't this job triggered only in case of success of the terraform-check one? I wasn't expecting to find a run_if here (but on the other yes)

Reply:
Oh, I see. It looks like these triggers are not supported in the config, but it appears that image jobs may support them. We may be able to add the fields manually into the job here.

Reply:
I have added/edited the fields manually.